playbooks/roles/openconnect/templates/instructions.md.j2

```diff
@@ -106,6 +106,11 @@ OpenConnect is an extremely high-performance and lightweight VPN server that als
 1. If this is your first time using AnyConnect, you will need to accept the Connection Request dialog that Android displays.
 1. You should be good to go! You can verify that your traffic is being routed properly by [looking up your IP address on Google](https://encrypted.google.com/search?hl=en&q=ip%20address). It should say *Your public IP address is {{ streisand_ipv4_address }}*.
 
+Some users have reported that AnyConnect prompts for a username and password. This is a bug; please report it. If this happens to you, use these credentials as a workaround:
```

Comment:
Who should the bug be reported to? Streisand? AnyConnect? Can you make the text a link to the correct repositories's new issue link?